depends/patches/qt/fix_montery_include.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/qtbase/src/plugins/platforms/cocoa/qiosurfacegraphicsbuffer.h b/qtbase/src/plugins/platforms/cocoa/qiosurfacegraphicsbuffer.h
```

Comment:
Can you add a comment here, the same as our other qt patches. Linking to the upstream commit.

Reply:
Done!